Framework/include/ScaleFactors.h

```diff
@@ -556,32 +553,18 @@ class ScaleFactors
         // For 2016 and 2018: Grab the individual pileup weight from the histogram found in PileupHistograms_*.root
         // For 2017: Using the puWeight stored in the nTuples
         // ----------------------------------------------------------------------------        
-        const auto& puWeight  = tr.getVar<double>("puWeight");
-        const auto& puSysUp   = tr.getVar<double>("puSysUp");
-        const auto& puSysDown = tr.getVar<double>("puSysDown");
-        const auto& tru_npv   = tr.getVar<double>("TrueNumInteractions");
+        const auto& puWeightUnCorr  = tr.getVar<double>("puWeight");
+        const auto& puSysUpUnCorr   = tr.getVar<double>("puSysUp");
+        const auto& puSysDownUnCorr = tr.getVar<double>("puSysDown");
```

Comment:
Are these variables stored in the new UL nTuples? I thought they made all new variables floats?

Reply:
They are in, presumably stored as `float`

Reply:
Ok let's change the types when we get a chance.

Reply:
Does it make sense that the above code just works without specifying `float` ?